new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -0,0 +1,1007 @@
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(6) List<String> namespaces) {
+
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(5, MINUTES).await();
+
+    // get a unique operator namespace
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Get a unique namespace for WebLogic domain1");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain1Namespace = namespaces.get(1);
+
+    logger.info("Get a unique namespace for WebLogic domain2");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domain2Namespace = namespaces.get(2);
+
+    // get a unique monitoring namespace
+    logger.info("Get a unique namespace for monitoring");
+    assertNotNull(namespaces.get(3), "Namespace list is null");
+    monitoringNS = namespaces.get(3);
+
+    // get a unique webhook namespace
+    logger.info("Get a unique namespace for webhook");
+    assertNotNull(namespaces.get(4), "Namespace list is null");
+    webhookNS = namespaces.get(4);
+
+    // get a unique NGINX namespace
+    logger.info("Get a unique namespace for NGINX");
+    assertNotNull(namespaces.get(5), "Namespace list is null");
+    final String nginxNamespace = namespaces.get(5);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domain1Namespace,domain2Namespace);
+
+    //install monitoring exporter
+    installMonitoringExporter();
+
+    // create and verify WebLogic domain image using model in image with model files
+    String imageName = createAndVerifyDomainImage();
+
+    // create and verify one cluster domain
+    logger.info("Create domain and verify that it's running");
+    createAndVerifyDomain(imageName, domain1Uid);
+
+    // get a free node port for NGINX
+    nodeportshttp = getNextFreePort(30305, 30405);
+    int nodeportshttps = getNextFreePort(30443, 30543);
+
+    // install and verify NGINX
+    nginxHelmParams = installAndVerifyNginx(nginxNamespace, nodeportshttp, nodeportshttps);
+    // create ingress for the domain
+    logger.info("Creating ingress for domain {0} in namespace {1}", domain1Uid, domain1Namespace);
+    Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+    clusterNameMsPortMap.put(clusterName, managedServerPort);
+    ingressHostList =
+        createIngressForDomainAndVerify(domain1Uid, domain1Namespace, clusterNameMsPortMap);
+
+  }
+
+  @Test
+  @DisplayName("Install Prometheus, Grafana , Webhook, Coordinator and verify WebLogic metrics")
+  public void testCheckMetrics() throws Exception {
+
+    createPvAndPvc("prometheus");
```

Comment:
Can we make the method createPvAndPvc() as action method in TestAction class.  I see similar local methods in ItDomainInPV class

Reply:
we have test in commontestutils, it takes v1pv and pvc params, the pv and pvc created in this test are specific to prom and grafana, so I think it is ok to keep in test class, please let me know